src/installing-from-source.rst

```diff
@@ -28,6 +28,9 @@ If you want to ensure you have the latest version of all submodules, preserving
 
    git submodule foreach --recursive git pull
 
+.. note::
+   If your Linux system is using Wayland rather than X11, the default watchers will not work. Read :ref:`window and idle watchers for Wayland<wayland-watchers>` for replacement modules supporting Wayland.
+
```

Comment:
```suggestion
```

I think we've made it clear enough elsewhere in docs.

Reply:
I think users will click the "install from source" link without scrolling down and expect complete instructions to be provided there. Having an additional step for Wayland users in the bottom of the "getting started" page is less intuitive I think. I removed it for now and if you agree with my rationale I'll add it back.